atlas/packages/app/src/components/VideoGallery.tsx

```diff
@@ -0,0 +1,127 @@
+	},
+	{
+		title: "Sample Video Title",
+		channel: "Channel Name",
+		showChannel: true,
+		views: "345k",
+		time: "2 weeks ago",
+		poster:
+			"https://s3-alpha-sig.figma.com/img/4baa/8670/e77491e871584e97b955d728627020f3?Expires=1593388800&Signature=LkSz8F8wftV81dxfumpYE5CToghiq5a3hvwywrhwVAXFwQvNzfymCImBCY9wIxILX4nnOlaNX0g~~iZ2RXD104VvLn6q967Hlg0uAH1h3eutIZzb4sQ0i4R-wXUw5ZNy2QjLCOIlKuCXMXSO1UiI8t8CJtdel9XaSqm1u~JsPFhRjRlO60RCi2cM2pJ~8G~kofCMT6J3NMgyRruVLUDY9FPGh5dI5XRFmeKbMThO2IT5KfUGsPDLb98i8gBrkLe2EKMx2DxHlDxcA3j2LVtf4yaGG-EmeDrQflknjdoAY0BKfC8Mob10ldGsYnJsMHOraz971POWOVpWTuoF0h6nNg__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA",
+	},
+	{
+		title: "Sample Video Title",
+		channel: "Channel Name",
+		showChannel: true,
+		views: "345k",
+		time: "2 weeks ago",
+		poster:
+			"https://s3-alpha-sig.figma.com/img/e0d6/4163/147cd1502ba0a99ea7258f31be9812b3?Expires=1593388800&Signature=av5e4oaaYXVDxA9U0DAQKuwA-vyC4VRaGYz2lO5IIaz-CP55B717sXn-V6H6JzMNZ7JavA9q8~uMK-r9jd-yxoia1UopEiUeSkZgQlbBhd4DEVBPoMhMF3nC--mZ4bQnVvK9xN933TCm9z46jkAOZ7U8UYp5TlA829Rnmgw3QkaNlPLW5UkgU04k6pAae5Y3t9pAnwErmDCDjHuaXKrCjCuG0zwk84f0JQAcZt4VojD8kVDFCf0IOHdZtIQMUk47CQfdWJ2KDBkm1oOcVAweTSHXdwzpiFQ~o1i-hZVYtgzIiGTE6IqW3qsksqiLtWwMsui2dvWXVu2actRRwR~OTg__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA",
+	},
+	{
+		title: "Sample Video Title",
+		channel: "Channel Name",
+		showChannel: true,
+		views: "345k",
+		time: "2 weeks ago",
+		poster:
+			"https://s3-alpha-sig.figma.com/img/63ab/b514/2a10cf17ceaf00972e7c12c18cacd832?Expires=1593388800&Signature=WRk44Vxkc1mYbXSdGroHP8Hvd4dfhtWMukS~2yfNlY6KSb94ZzeFaKID8dMj4wznZo-sSp~lAagV7NJMFO9QLCEw0E7YeX3TnOWeFDrx63wkc388h-srSxxrNVUpB6WUCfkPtdusDrfKOx2HvM6zZLlSgYs4-McX2VTJ-g7bKx2Pzt64mox6wTGlbXWREkV2sbLSBSF6P0fpwge3VPt5jX1E~vDJ8X4mtZgN1DRIQMtc2dfZhZhyL1jNeIZe5WJyOcY9b2E3KL54pWt13UGsI08nTzloqOPFAt~fHwbwUbwWeelbjkzSABh55KYVNePerISUIy~esGJ11px7l9oWTw__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA",
+	},
+	{
+		title: "Sample Video Title",
+		channel: "Channel Name",
+		showChannel: true,
+		views: "345k",
+		time: "2 weeks ago",
+		poster:
+			"https://s3-alpha-sig.figma.com/img/b327/39be/143853e8ca345927ee54a7ca9acaed0c?Expires=1593388800&Signature=G1vikiLUBwdBdrzS3RC1WScSJSAAD7FEteNX4bfPf0VQL0w9fsMfha31S-l6EHxU~WTh3GAic8X5furwHjwLbE2nVUIiA0q6Rizxfd8kT97Z-9uQzPbCkla6a1qIX-7wq19t2evAVST--Sz7wfEYVCUTdZaEmqgBsJnYI3fl7nzrwLhaaTTN3liOTltgvlQn8btRClliTWENyHBwu6cwvMHuX6ePt5swhIj4gz34F0MyY~PtnsFbBHFerShd3t2DYsMqDv8CEB089v6IYZh87Rnqk7jJFZC2ZBULhjlwKQpboJOI35cov3aGOnuTe5vcmULLSfEXuV5kWJrtzJnQlA__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA",
+	},
+];
+
+const articleStyles = css`
+	max-width: 320px;
+	margin: auto ${theme.spacing.m};
+`;
+
+export default function VideoGallery({ title = "" }: VideoGalleryProps) {
+	const videos = videoPlaceholders.concat(videoPlaceholders).concat(videoPlaceholders);
+	const [controlsTop, setControlsTop] = useState<SerializedStyles>(css``);
+	const imgRef = useCallback((node: HTMLImageElement) => {
+		if (node != null) {
```

Comment:
We should probably never do loose equality comparisons and we should have an ESLint to enforce this.

We can also do a broader check than just for `null` here with `if (node)`

Reply:
I agree on adding an ESLint rule to check for loose equality check, the `null` comparison here is very safe though, since `x == null` will return `true` iff `x === undefined || x === null` and there is no type coercion involved

Reply:
I agree, here it shouldn't do anything weird, but I think it's a good rule to use strict equality wherever we can.